### PR TITLE
add Workflow Github_Pull_Request_Review_Events_to_Slack

### PR DESCRIPTION
**Pull request by nkumarasamy-r7**
## Requirements

Developers, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow/Snippet contains title and detailed description
- [ ] Workflow/Snippet title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] User friendly step names in [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] User friendly step descriptions where needed
- [ ] Workflow contains at least 1 artifact with well formatted information
- [ ] Directory created and named after the title the snippet or workflow e.g. `workflows/Basic_Domain_Enrichment_Report/`, `snippets/Domain_Enrichment`
- [ ] Workflow `.icon` file is in the workflow's directory e.g. `workflows/Basic_Domain_Enrichment_Report/Basic_Domain_Enrichment_Report.icon`
- [ ] Take PNG screenshots of the workflow or snippet e.g.  https://market.komand.com/workflows/ross/chatops-github-commands/1.0.0, https://market.komand.com/snippets/jschipp/basic-domain-enrichment-report/0.1.0
- [ ] Create a directory called `screenshots` in your workflow's/snippet's directory
- [ ] Includes PNG screenshots of the workflow in canvas, job, artifact in the `screenshots` directory e.g.  `workflows/Basic_Domain_Enrichment_Report/screenshots/`
- [ ] PNG screenshot file names should conform to style e.g. `workflow1.png`, `artifact1.png`, `job.png`
- [ ] Workflow verified as working through WUI
- [ ] Credentials are not exported and stored in git
- [ ] `workflow.spec.yaml` file with metadata accompanies workflow
- [ ] Workflow documentation is provided in `help.md`
- [ ] Directory structure is followed, see https://github.com/rapid7/insightconnect-workflows/blob/master/README.md

### Assessment

Post your results with screenshots in this PR using the following style:

```
### UI

Screenshots of the workflow being used in Komand for validating use (drag and drop screenshots).

#### Workflow Builder

<img width="697" alt="statsd_workflow" src="https://user-images.githubusercontent.com/2321183/27349935-8e00fd12-55be-11e7-9d86-31250743dc32.png">

#### Job

<img width="1022" alt="screen shot 2017-06-20 at 2 03 31 pm" src="https://user-images.githubusercontent.com/2321183/27350999-9ef49fe0-55c1-11e7-9a74-7b31927d4d9a.png">

#### Artifact

<img width="937" alt="screen shot 2017-06-20 at 2 03 41 pm" src="https://user-images.githubusercontent.com/2321183/27351011-a4f41880-55c1-11e7-89fa-0e671221863e.png">
```

### CODE REVIEW

File: workflows/Github_Pull_Request_Events_to_Slack/help.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Technical Details
+
+Plugins utilized by workflow:
+
+|Plugin|Version|Count|
+|----|----|--------|
+Python 2 Script 2.0.3
+
+
+
```
Comment (lmilby-r7) [inline workflows/Github_Pull_Request_Events_to_Slack/help.md]: Remove the 2 extra lines

File: workflows/Github_Pull_Request_Events_to_Slack/help.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+API Key 
+
+
+# Documentation
+
+## Setup
+
+Once the workflow has been downloaded, login to InsightConnect and “Import” it into the workflow builder. 
+
+
```
Comment (lmilby-r7) [inline workflows/Github_Pull_Request_Events_to_Slack/help.md]: Remove extra line

File: workflows/Github_Pull_Request_Events_to_Slack/help.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* Post Closed Message
+* Post Review Label Message 
+* Post Review Request Message
+* Post Open Message
+
+# Requirements
+
+API Key 
+
+
```
Comment (lmilby-r7) [inline workflows/Github_Pull_Request_Events_to_Slack/help.md]: Remove extra line

File: workflows/Github_Pull_Request_Review_Events_to_Slack/help.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+* Post Approved Message
+* Changes Requested Message
+* Other Message
+
+# Requirements
+
+API Key 
+
+
```
Comment (lmilby-r7) [inline workflows/Github_Pull_Request_Review_Events_to_Slack/help.md]: Remove extra line

File: workflows/Github_Pull_Request_Review_Events_to_Slack/help.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+## Technical Details
+
+Plugins utilized by workflow:
+
+|Plugin|Version|Count|
+|----|----|--------|
+Python 2 Script 2.0.3
+
+
+
```
Comment (lmilby-r7) [inline workflows/Github_Pull_Request_Review_Events_to_Slack/help.md]: Remove extra line

File: workflows/Github_Pull_Request_Review_Events_to_Slack/help.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+API Key 
+
+
+# Documentation
+
+## Setup
+
+Once the workflow has been downloaded, login to InsightConnect and “Import” it into the workflow builder. 
+
+
```
Comment (lmilby-r7) [inline workflows/Github_Pull_Request_Review_Events_to_Slack/help.md]: Remove extra line

File: workflows/Github_Pull_Request_Review_Events_to_Slack/help.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# Description
+
+This workflow is designed to notify Github Pull Request Review Events to Slack.  Post messages directly into your Slack workspace in your preferred channels, private groups, or IM channels.
```
Comment (lmilby-r7) [inline workflows/Github_Pull_Request_Review_Events_to_Slack/help.md]:
```suggestion
This workflow is designed to notify Github Pull Request Review Events to Slack. Post messages directly into your Slack workspace in your preferred channels, private groups, or IM channels.
```

File: workflows/Github_Pull_Request_Review_Events_to_Slack/workflow.spec.yaml [hunk 8]
```diff
@@ -0,0 +1,18 @@
+extension: workflow
+products: ["insightconnect"]
+name: Github_pull_request_review_events_to_slack
+title: Github_pull_request_review_events_to_slack
+description: "This workflow is designed to automatically analyze incoming emails to determine if they are malicious. This workflow assumes a user is using their report
```
Comment (lmilby-r7) [inline workflows/Github_Pull_Request_Review_Events_to_Slack/workflow.spec.yaml]: shorter description if possible. More detail in the help.md and high level in the spec.

File: workflows/Github_Pull_Request_Events_to_Slack/workflow.spec.yaml [hunk 9]
```diff
@@ -0,0 +1,20 @@
+extension: workflow
+products: ["insightconnect"]
+name: Github_pull_request_events_to_slack
+title: Github_pull_request_events_to_slack
+description: This workflow is designed to notify Github Pull Request Events to Slack.  Post messages directly into your Slack workspace in your preferred channels, private groups, or IM channels.
```
Comment (lmilby-r7) [inline workflows/Github_Pull_Request_Events_to_Slack/workflow.spec.yaml]: shorter description if possible. More detail in the help.md and high level in the spec.

**Comment by jschipp-r7** [comment]
We'll need to hold on this until the platform allows the api key in the query param.

**Pull request closed without merging**